backend/src/services/release.ts

```diff
@@ -295,6 +335,142 @@ export async function getReleaseBySemver(user: UserInterface, modelId: string, s
   return release
 }
 
+function getSemverQueryBounds(querySemver: string) {
```

Comment:
Could we change this to something like `parseSemverQuery()`?

Reply:
Changed